### PR TITLE
feat: add security insights plugin to marketplace

### PR DESCRIPTION
**Pull request by lewtakm**
## Security Insights Plugin

Hello,
I would like to add new plugin to the marketplace. 
This plugin shows insights from GitHub Code Scanning inside Backstage app. 
Right now plugin could work as a standalone tab and widget placed in Overview page. 
Plugin also supports dismissing open security alerts directly from Backstage. 

Plugin repository: [Security Insights](https://github.com/RoadieHQ/backstage-plugin-security-insights)

Any suggestions and feedback how can the plugin could be improved would be more than welcome.

**Screenshots:**

**Tab:**
chrome_9VKuKYrIVw

**Widget:**
chrome_wIlofLHL5t

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/spotify/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)

### CODE REVIEW

File: microsite/data/plugins/security-insights.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Security Insights
+author: roadie.io
+authorUrl: https://roadie.io/
+category: Monitoring
```
Comment (stefanalund) [inline microsite/data/plugins/security-insights.yaml:5]: I think this should be in a Security category.

**Comment by goober** [comment]
This looks really promising. We are using [SonarQube](https://www.sonarqube.org/) for our static code analysis.
Does the Security Insights plugin provide an interface that we can leverage to implement support for other security insights providers or is it mainly for github?

**Comment by dtuite** [comment]
Hi @goober I work with @lewtakm. Great idea regarding SonarQube! A few people have asked for that already and we going to look into that next. Is there any chance I could convince you to take a call with us so we can learn a bit more about how you use SonarQube in your company? I'll send you an email now.

@stefanalund this plugin works natively with just a GitHub integration and I know security will be a big use case in the future. Do you think it makes sense to add it to the sample app?

**Comment by andrewthauer** [comment]
@dtuite - We've also started using SonarQube, so I'd be interested in joining the discussion as well if that's ok?

**Review by benjdlambert — APPROVED** [review]

**Comment by dtuite** [comment]
We have a call set up at 2pm UTC time on Tuesday 27th to talk about what a SonarQube plugin would look like and what it should do. Email me at david@roadie.io if you would like to join.

**Pull request merged**